src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -217,9 +217,14 @@ class IOUAmountPage extends React.Component {
                     styles.justifyContentCenter,
                 ]}
                 >
-                    <TouchableOpacity onPress={() => Navigation.navigate(this.props.hasMultipleParticipants
-                        ? ROUTES.getIouBillCurrencyRoute(this.props.reportID)
-                        : ROUTES.getIouRequestCurrencyRoute(this.props.reportID))}
+                    <TouchableOpacity onPress={() => {
+                        if (this.props.hasMultipleParticipants) {
+                            return Navigation.navigate(ROUTES.getIouBillCurrencyRoute(this.props.reportID));
+                        }
+                        return Navigation.navigate(this.props.iouType === CONST.IOU.IOU_TYPE.SEND
```

Comment:
NAB (Not a Blocker): I'd somewhat prefer -

```
if (x) return abc
if (y) return def
return ghi
```

Reply:
Resolved.